### PR TITLE
Dummy commit to force dev-v2.5 to regenerate

### PR DESCRIPTION
**Pull request by aiyengar2**
Triggers a `make stage-release` push workflow to regenerate all of released/ + index.yaml after the below PR is merged.

Related PR: https://github.com/rancher/charts/pull/1167

### CODE REVIEW

**Review by nickgerace — APPROVED** [review]
LGTM after https://github.com/rancher/charts/pull/1167
Though @aiyengar2 would this be a good opportunity for any docs or README changes instead?

**Review by cbron — APPROVED** [review]
THis is fine, but you could probably just run last build CI again

**Comment by aiyengar2** [comment]
Yeah @cbron before merge I'll use Github to update the branch, which will automatically re-run CI.

**Pull request merged**
